### PR TITLE
paramminer: Add ParamGuesser options and functionality

### PR DESCRIPTION
**Pull request by ArkaprabhaChakraborty**
Related to zaproxy/zaproxy#7140
Signed-off-by: ArkaprabhaChakraborty <chakrabortyarkaprabha998@gmail.com>

### CODE REVIEW

File: addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties [hunk 1]
```diff
@@ -24,11 +24,18 @@ paramminer.toolbar.progress.select  = --Select Scan--
...
+paramminer.dialog.predefined.header_wordlists = Use Predefined Header Wordlists
+paramminer.dialog.predefined.cookie_wordlists = Use Predefined Cookie Wordlists
+
+paramminer.dialog.custom.url_wordlists = Use Custom Url Wordlists
+paramminer.dialog.custom.header_wordlists = Use Custom Header Wordlists
+paramminer.dialog.custom.cookie_wordlists = Use Custom Cookie Wordlists
+
 paramminer.dialog.skip_boring_headers = Skip boring headers
 paramminer.dialog.threadpool_size = Threadpool Size:
 paramminer.dialog.error.url = URL is required
```
[the remark this answers is not in the document]
Comment (ArkaprabhaChakraborty) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties:41]: I mean it's just there to enhance readability... kind of easy on the eyes.

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties:41]: I'm driving toward conformance with: https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/#style-guidelines

To me there are only 3 options. Title Caps, use periods, use colons.

[the remark this answers is not in the document]
Comment (ArkaprabhaChakraborty) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: Soooo What do I do about this? :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (ArkaprabhaChakraborty) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: well these will be accompanied by checkboxes so I guess a ":" would be good :)

[the remark this answers is not in the document]
Comment (ArkaprabhaChakraborty) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: what do you say @kingthorin ?

[the remark this answers is not in the document]
Comment (ArkaprabhaChakraborty) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: all of these staring from cache buster to boring headers are check box labels

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: Thanks, that seems better.

[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: x2

[the remark this answers is not in the document]
Comment (ArkaprabhaChakraborty) [reply addOns/paramminer/src/main/resources/org/zaproxy/addon/paramminer/resources/Messages.properties]: Should be done now

**Pull request merged**